packages/app-page-builder/src/admin/plugins/pageDetails/header/pageOptionsMenu/PageOptionsMenu.tsx

```diff
@@ -15,6 +16,7 @@ import { setHomePage } from "./graphql";
 import { useConfirmationDialog } from "@webiny/app-admin/hooks/useConfirmationDialog";
 import { getPlugins } from "@webiny/plugins";
 import { PbPageDetailsHeaderRightOptionsMenuItemPlugin } from "@webiny/app-page-builder/types";
+import { useConfigureDomainDialog } from "@webiny/app-page-builder/utils/useConfigureDomain";
```

Comment:
Instead of `/app-page-builder/utils/`, put it into the `admin/hooks` folder, where all other hooks are located as well.